src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -276,7 +291,7 @@ struct GeneralizedHarmonicGroup {
  * *rolled-on* to) at the beginning of an evolution. This parameter sets
  * the coordinate time at which roll-on begins.
  */
-struct GaugeRollOnStart : Tags::GaugeHRollOnStartTime {
+struct GaugeRollOnStart {
   using type = double;
   static std::string name() noexcept { return "GaugeRollOnStart"; }
```

Comment:
You don't have to specify a `name` if it is coincident with the `struct` name.

Reply:
Yes - but I have a mild preference for keeping it explicit for aesthetic completeness and easier readibility for newcomers, unless there's actually a runtime performance hit.

Reply:
(Not a binding request, just an opinion:) The reason I prefer implicit names is that you can't screw them up with typos or, more commonly, by forgetting to change them when the struct name changes (such as when the struct is copy-pasted as a template for a new tag struct).

Reply:
I agree with @wthrowe

Reply:
Okay, this is convincing enough for me, and so I've removed `name`s from OptionTags where it is redundant.